### PR TITLE
docs: add action/output page

### PR DESCRIPTION
**Pull request by sergkudinov**

### CODE REVIEW

File: docs/content/action/output.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+navtitle: Output
+sort: 3
+---
+
+# Action output
+
+Action output is the result returned by Nikita's actions. It is defined by the [action handler](/current/action/handler#return) enriched with the default values.
```
Comment (wdavidw) [inline docs/content/action/output.md:8]: It is the value return by an action handler eventually modified by plugins and wrapped inside a promise

File: docs/content/action/output.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+- `status` - a boolean value of the [action status](/current/usages/status).
+- `log` - an array of objects with [logging](/current/usages/logging_debugging) information of the action execution.
+
+Optionally, enriching the handler return value can be disabled with the [`raw_output` metadata](/current/metadata/raw_output).
```
Comment (wdavidw) [inline docs/content/action/output.md:13]: Use the `raw_output` metadata to instruct plugins to not alter the output (this deserve its own section with an example)

**Review by wdavidw — CHANGES_REQUESTED** [review]

File: docs/content/action/output.md (hunk 2 above)
Comment (sergkudinov) [reply docs/content/action/output.md:13]: I am referencing  to page where will be this example

File: docs/content/action/output.md (hunk 1 above)
Comment (sergkudinov) [reply docs/content/action/output.md:8]: > by plugins
Where we describe what is Nikita plugin?

**Review by wdavidw — APPROVED** [review]
I'll finish locally

**Pull request merged**